test/logic_value_test.dart

```diff
@@ -2052,11 +2052,30 @@ void main() {
   group('RadixString', () {
     test('radixString roundTrip', () {
       final lv = LogicValue.ofBigInt(BigInt.from(737481838713847), 61);
+      LogicValue.ofRadixString(lv.toRadixString());
```

Comment:
what's the point of these new lines?  aren't they redundant with the loop below?

Reply:
Those were for debug and will be deleted.